server/src/test/java/org/elasticsearch/index/IndexSortSettingsTests.java

```diff
@@ -135,13 +141,8 @@ public void testInvalidMissing() {
             " must be one of [_last, _first]"));
     }
 
-    public void testIndexSorting() {
+    public void testIndexSortingNoDocValues() {
```

Comment:
I just renamed this method. I didn't change what it does. All of the removed lines are just relocated into `buildIndexSort`.